docs/source/ADMINISTRATA.rst

```diff
@@ -119,6 +119,29 @@ We use the :term:`Azure` and :term:`GitHub Actions` cloud services for
 .. |.github/workflows/nix.yml| replace::    :file:`{FiPySource}/.github/workflows/nix.yml`
 .. _.github/workflows/nix.yml: https://github.com/usnistgov/fipy/blob/master/.github/workflows/nix.yml
 
+===============
+Conda Lockfiles
+===============
+
+The `conda-lock <https://github.com/conda/conda-lock>`_ lockfiles in
+:file:`environments/locks/` can be updated with::
+
+    $ for solver in petsc pysparse scipy trilinos
```

Comment:
Refactor as a rst code-block.

Reply:
It is?

"[A paragraph consisting of two colons ("::") signifies that the following text block(s) comprise a literal block. The literal block must either be indented or quoted (see below).](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks)